ir/shared/src/main/scala/org/scalajs/ir/Types.scala

```diff
@@ -266,7 +258,8 @@ object Types {
   sealed abstract class NonArrayTypeRef extends TypeRef
 
   /** Primitive type reference. */
-  final case class PrimRef private[ir] (tpe: PrimTypeWithRef)
+  final class PrimRef private[Types] (val tpe: PrimTypeWithRef)(
+      charCodeInit: Char, displayNameInit: String) // "Init" variants so we can have good Scaladoc on the val's
```

Comment:
Do we still need the two parameter lists? IIUC that was just for the case class?

Reply:
No, we don't. Merged.